ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -383,28 +351,28 @@ checks:
     rules:
       - 'c:mount -> r:\s/dev/shm\s && r:noexec'
 
-# 1.1.19 Disable Automounting
-  - id: 20521 
+# 1.1.23 Disable Automounting
+  - id: 20521
     title: "Disable Automounting"
     description: "autofs allows automatic mounting of devices, typically including CD/DVDs and USB drives."
     rationale: "With automounting enabled anyone with physical access could attach a USB drive or disc and have its contents available in system even if they lacked permissions to mount it themselves."
     remediation: "Run the following command to disable autofs: systemctl disable autofs"
     compliance:
-      - cis: ["1.1.19"]
+      - cis: ["1.1.23"]
       - cis_csc: ["8.4","8.5"]
       - pci_dss: ["2.2.4"]
       - nist_800_53: ["CM.1"]
       - tsc: ["CC5.2"]
     condition: none
     rules:
-      - 'c:systemctl is-enabled autofs -> r:enabled'
+      - 'p:autofs'
```

Comment:
Check if automounting is disabled, not running.
```
# systemctl show "autofs.service" | grep -i unitfilestate=enabled
```
See check 1.1.23 of the CIS benchmark for Amazon Linux 2 2.0.0